bin/foreman-proxy-certs-generate

```diff
@@ -28,5 +30,32 @@ Kafo::KafoConfigure.hooking.register_pre(:init) do
   org_param.value = organization
 end
 
+Kafo::KafoConfigure.hooking.register_pre(:boot) do
+  app_option(
+    '--certs-skip-check',
+    :flag,
+    "This option will cause skipping the certificates sanity check. Use with caution",
+    :default => false
+  )
+end
+
+Kafo::KafoConfigure.hooking.register_pre(:pre_commit) do
+  ca_file   = param('certs', 'server_ca_cert').value
+  cert_file = param('certs', 'server_cert').value
+  key_file  = param('certs', 'server_key').value
+
+  unless app_value(:certs_skip_check) || [cert_file, ca_file, key_file].all? { |v| v.to_s.empty? }
+    # TODO: branding for foreman-proxy
```

Comment:
@evgeni should we modify katello-certs-check to always accept `foreman-proxy` for `-t`, even if it's branded?

Reply:
Doesn't it do that? (I have no memories about how I implemented branding there)

Reply:
As I read this part I think it only accepts the branded value:
https://github.com/theforeman/foreman-installer/blob/8b7b58dfd4b5d50a391b6c21d9b8db777896fc7f/bin/katello-certs-check#L21